src/components/WalletAuthorizationComponent.jsx

```diff
@@ -318,6 +318,8 @@ class WalletAuthorizationComponent extends Component<Props, State> {
           ${address.slice(0, 10)}...${address.slice(-10)}`
         } else if (errors.setTokenAllowance === WalletErrors.ledger.contractDataDisabled) {
           errorInstruction = 'Please enable Contract data on the Ethereum app Settings'
+        } else if (errors.submitTx === WalletErrors.ledger.incorrectSigningKey) {
+          errorInstruction = 'Please make sure you have selected the correct Ledger network'
```

Comment:
${cryptoType} network instead of "ledger network"

Reply:
fixed